elyra/metadata/metadata_app.py

```diff
@@ -473,6 +473,96 @@ def start(self):
               ("is" if num_invalid_exported == 1 else "are") + " invalid)")
 
 
+class SchemaspaceImport(SchemaspaceBase):
+    """Handles the 'import' subcommand functionality for a specific schemaspace."""
+
+    directory_option = CliOption("--directory", name='directory',
+                                 description='The local file system path from where the metadata will be imported',
+                                 required=True)
+
+    replace_flag = Flag("--replace", name='replace',
```

Comment:
We might want to change this to `--overwrite` since we got rid of it in the `install` command (split into `create` and `update`) and pipeline export will use `--overwrite`.  I would suggest updating metadata export to use `--overwrite` as well.

cc: @ptitzler